docs/installation.rst

```diff
@@ -560,6 +561,156 @@ As mentioned before, the script engine can be started from within ``$SCRIPT_ENGI
 You have to ensure your Init script starts the Node.js executable ``node`` and uses the JavaScript file ``$SCRIPT_ENGINE_HOME/server.js`` as an argument.
 Note, if you use a restricted user to execute the command, make sure the user has appropriate access to the ``$SCRIPT_ENGINE_HOME`` directory.
 
+.. _install-mail-relay:
+
+Installing the Workflow Accelerator Mail Relay
+----------------------------------------------
+.. important:: You only need to install and configure the Mail Relay if you purchased a version of Workflow Accelerator which allows you to use email triggers.
+
+The mail relay is a SMTP server which receives emails and forwards them via HTTP to the Workflow Accelerator web application.
+It is required in order to start a workflow by email.
+If you don't plan on starting workflows by email, you don't have to setup this component and can skip this section.
```

Comment:
_to set this comment up_

As a verb, _set up_ is two words, and as a noun is hyphenated to _set-up_. There are more occurrences of this to fix, so I won't comment on all of them.